news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
rename file to all lower-case and add a proper date placeholder

we'll go with 17th, so

2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md